metrics/src/test/java/com/expedia/adaptivealerting/metrics/functions/service/graphite/GraphiteQueryServiceTest.java

```diff
@@ -60,8 +58,9 @@ public void initMetricSourceSinkConfigSetup() throws Exception {
         MockitoAnnotations.initMocks(this);
         val config = new TypesafeConfigLoader(APP_ID).loadMergedConfig();
         metricSourceSinkConfig = config.getConfig(METRIC_SOURCE_SINK);
-        val functionsInputFile = "/config/functions-test.txt";
-        metricFunctionsSpec = MetricFunctionsReader.readFromInputFile(functionsInputFile).get(0);
+        val functionsInputFile = "config/functions-test.txt";
+        metricFunctionsSpec = MetricFunctionsReader.readFromInputFile(
+                ClassLoader.getSystemResource(functionsInputFile).getPath()).get(0);
```

Comment:
don't understand this. you are passing ClassLoader in the test but in actual you are using 
```val input_file = INPUT_FILE_PATH + INPUT_FUNCTIONS_FILENAME;
MetricFunctionsReader.readFromInputFile(input_file)```

Reply:
Deployment uses absolute path of file - "/config/functions.txt". 
Updated test cases to reflect the same as well.